code-style/git/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,82 @@
+### Description
+
+_[add description of work done here]_
+
+### Spec
+Designs: _[link to design if applicable]_
+
+See Story: _[/[ISSUE-NUMBER]_
+
+### Validation
+_[delete anything irrelevant to this PR]_
+
+* [ ] This PR has visual elements, so it was reviewed by a designer.
+* [ ] This PR has code changes, and our linters still pass.
+* [ ] This PR effects production code, so it was browser tested (see below).
+* [ ] This PR has new code, so new tests were added or updated, and they pass.
+* [ ] This PR has new SCSS functions, mixins, or variables, so [Sass Tests](https://seesparkbox.com/foundry/how_and_why_we_unit_test_our_sass) were added or updated, and they pass.
+* [ ] This PR has copy changes, so copy was proofread and approved.
+* [ ] The content of this PR requires documentation, so we added a detailed description of component purpose, requirements, quirks, and instructions for use by designers and developers. Along with accessibility information if pertinent.
+
+#### To Validate
+  1. Make sure all PR Checks have passed (CircleCI, Code Climate, Snyk, etc).
+  2. Pull down all related branches.
+  3. Confirm all tests pass.
+  4. Navigate to... _[continue instructions here]_  
+
+  _[For an example of good validation instructions, check out [Bryan's Bouncy Ball PR](https://github.com/sparkbox/bouncy-ball/pull/56#issue-192153701).]_
+
+---
+
+#### Browser Testing
+_[delete if irrelevant to this issue]_
+
+#### Gold Level Support
+**macOS**
+* [ ] Safari 9+ (latest subversion)
+* [ ] Chrome (latest version)
+* [ ] Firefox (latest version)
+
+**Windows**
+* [ ] IE 11+
+* [ ] Chrome (latest version)
+* [ ] Firefox (latest version)
+* [ ] Edge 16
+
+**Mobile**
+
+* [ ] Safari 9+
+* [ ] Android 6+
+* [ ] Samsung Browser (latest version)
+* [ ] Chrome (latest version)
+
+#### Silver Level Support
+
+In these browsers, behavior & design are acceptable, but may not be as originally intended. A user is still able to access all content and functionality as described.
+
+* [ ] IE 10 on Windows
```

Comment:
wonder if there's a vague reference here to something like "previous version?"

Reply:
I'm not sure I understand.

Reply:
I didn't do a great job of explaining why I was suggesting "latest version." I tried to explain that a bit better in the last comment. But I was trying to think of the longevity of this example and not needing to update it often. So I was trying to recommend ways to make vaguer references to the philosophy behind the browsers/devices referenced vs. specifying exact versions. I'm open. Maybe what we need is just to explicitly state that this is an example from a project in mid-2018 and that browsers/device support levels should be determined based on analytics.

Reply:
I think I can make that work!!